utils/logger.js

```diff
@@ -0,0 +1,109 @@
+const createLogger = () => {
```

Comment:
Directory name is `utils`, but isn't this more a service than a utility?

Reply:
Valid point! I couldn't think of where to put it honestly 🤔 I'm not sure I want to suffix it with 'service' but it makes more sense than Utils